src/app/project/controllers/collect-records.ctrl.js

```diff
@@ -404,13 +447,30 @@ angular.module('app.project').controller('CollectRecordsCtrl', [
       });
     };
 
+    $scope.tableControl.getFilteredRecordsCount = function() {
+      return (
+        $scope.tableControl.records &&
+        `${$scope.tableControl.records.length}/${collectRecordsCount}`
```

Comment:
What happens when `collectRecordsCount` is 0?

Reply:
if collect record 0 or table control records is 0, it's a falsy, the filter label won't display anything.